lib/offline/download_manager.js

```diff
@@ -168,6 +168,60 @@ shaka.offline.DownloadManager = class {
     return newPromise;
   }
 
+  /**
+   * Add a request to be downloaded as part of a group.
```

Comment:
I don't think this is accurate. This method doesn't actually download anything, right? It's just storing already-downloaded data.

Reply:
Actually it is self-generated data, I will update the comment.